extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -47,14 +47,14 @@ public function awardCustomNotInTrackBadge( $user, $badge_type_id ) {
 		global $wgExternalSharedDB;
 
 		$achConfig = AchConfig::getInstance();
-		if ( !$achConfig->shouldShow( $badge_type_id ) ) {
+		if( !$achConfig->shouldShow( $badge_type_id ) ) {
 			wfProfileOut( __METHOD__ );
 			return;
 		}
 
 		$this->mUser = $user;
 
-		if ( self::canEarnBadges( $this->mUser ) ) {
+		if( self::canEarnBadges( $this->mUser ) ) {
 
 			$where = array( 'badge_type_id' => $badge_type_id, 'user_id' => $this->mUser->getId() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "user_id" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "badge_type_id" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)